Doc/library/random.rst

```diff
@@ -294,6 +294,10 @@ be found in any statistics text.
 
    Return the next random floating point number in the range ``0.0 <= X < 1.0``
 
+   .. warning::
+      Calling this function in the concurrent use can cause performance degradation.
```

Comment:
`in the concurrent use` is not grammatical.  Either `in the concurrent use case` or `in concurrent uses` or `in multiple threads`

Reply:
Sam's suggestion is even better.